Dockerfile

```diff
@@ -1,26 +1,21 @@
-FROM rhel7
+# Using playbook2image as a base
+# See https://github.com/aweiteka/playbook2image for details on the image
+# including some of the settings below
+FROM docker.io/aweiteka/playbook2image:latest
 
-MAINTAINER Troy Dawson <tdawson@redhat.com>
+# The playbook to be run is specified via the PLAYBOOK_FILE env var
+# Setting a default of openshift_facts.yml as it's an informative playbook
+# that can help test that everything is set properly (inventory, sshkeys)
+ENV PLAYBOOK_FILE=plabyooks/byo/openshift_facts.yml \
+    ANSIBLE_HOST_KEY_CHECKING=False \
```

Comment:
I get why this is needed for non-interactive usage, but could there be a comment as to why?

Something to the effect that unless you mount in a pre-verified known_hosts file, there's no security mechanism to verify the host keys at the time this is running. While we could use the method in http://stackoverflow.com/a/35213839/262768 to set up such a file at run-time, it wouldn't offer any actual security.